examples/compare_exp_results.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+EXP_OUT_DIR=$1
+OUT_DIR=$2
+
+function get_vcfs_md5 () {
+    grep -vh '^#' ${1}/*.vcf | sort -s | md5sum
```

Comment:
see earlier comment about detecting the available tool and potentially using `md5 -r [files...]`.

Reply:
Now using whichever command is available.